CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 ### 🐞 Fixed
 - Fix support for markdown ordered list with all numbers [#3090](https://github.com/GetStream/stream-chat-swift/pull/3090)
 - Fix support for markdown italic and bold styles inside snake-styled text [#3094](https://github.com/GetStream/stream-chat-swift/pull/3094)
+- Fix markdown-formatter customization accessibility [#3101](https://github.com/GetStream/stream-chat-swift/pull/3101)
```

Comment:
We should put this into ✅ additions and name it "Add support markdown styling customization"